src/common.ts

```diff
@@ -29,6 +29,7 @@ export const DEPRECATED_GRAPHQL_WS_PROTOCOL = 'graphql-ws';
 export enum CloseCode {
   InternalServerError = 4500,
   InternalClientError = 4005,
+  BadGateway = 1014,
```

Comment:
These are application specific close codes, close codes that are used by the GraphQL over WS spec.

```suggestion
```